### DISCUSSION TITLE
Update screen showing in a large width and height

### DISCUSSION THREAD

**Discussion by MarJose123** [beyondcode/herd-community#942] [Ideas]
### Platform

Windows

### Operating system version

Windows 11

### System architecture

Intel (x86)

### Herd Version

1.9

### Bug description

Update Screen 

![image](https://github.com/user-attachments/assets/512f0eb0-5f58-4e57-93ee-d87d46e2d7de)

**Comment by MarJose123** [comment]
In addition, my machine was forced to restart after the update.

**Comment by sschlein** [comment]
I am not sure that I understand the issue

**Comment by MarJose123** [comment]
With the screen size, I think it's not an issue, more on a suggestion to make it smaller?

With regards to the issues that my machine was forced restart after updating to `v1.10` not sure if it was a bug or not.
Context: I've update my herd, and during the update it restart my machine without a prompt that the machine will restart.

**Comment by sschlein** [comment]
I can't come up with a reason why Herd would restart your machine, there's nothing in the installer that would do that – so I guess that was something else 🤔

I'm moving this into the discussions tab as a feature idea for a smaller update window and to see if more people face the restart issue.
